setup.py

```diff
@@ -50,7 +50,7 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        "checking": ["black", "hacking", "isort", "mypy==0.782"],
+        "checking": ["black", "hacking", "isort", "mypy==0.782", "blackdoc"],
```

Comment:
[Note] The license of `blackdoc` is the [MIT](https://github.com/keewis/blackdoc/blob/master/LICENSE) license. I think we can use it in terms of the license.